packages/config/categories.ts

```diff
@@ -4,7 +4,7 @@ export type CategoryInfo = {
 	description: string;
 };
 
-export type CategoryKeys = 'codeQuality' | 'css' | 'db' | 'testing' | 'additional';
+export type CategoryKeys = 'codeQuality' | 'css' | 'db' | 'testing' | 'additional' | 'baas';
```

Comment:
I'd put in under DB rather than creating a new category 
```suggestion
export type CategoryKeys = 'codeQuality' | 'css' | 'db' | 'testing' | 'additional';
```